Editor/Window/UserProfileCreation.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AmazonGameLiftPlugin.Core.Shared;
+using Editor.CoreAPI;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace AmazonGameLift.Editor
+{
+    internal class UserProfileCreation
+    {
+        private readonly VisualElement _container;
+        private BootstrapSettings _bootstrapSettings;
+        private CancellationTokenSource _refreshBucketsCancellation;
+        private StateManager _stateManager;
+        private AwsUserProfilesPage _profilesPage;
+        
+        
+        public UserProfileCreation(VisualElement container, StateManager stateManager, AwsUserProfilesPage profilesPage)
+        {
+            _container = container;
+            _stateManager = stateManager;
+            _profilesPage = profilesPage;
+        }
+        
+        public void BootstrapAccount(string bucketName)
+        {
+            var bucketResponse = CreateBucket(bucketName);
+            if (bucketResponse.Success)
+            {
+                _container.Q<VisualElement>(null, "Tab2Success").style.display = DisplayStyle.Flex;
+                _stateManager.IsBootstrapped = true;
+                _profilesPage.UserProfileSelection.BucketSelection(bucketName);
+            }
+            else
+            {
+                _stateManager.IsBootstrapped = false;
+                var errorBox = _container.Q<VisualElement>("Tab2Error");
+                errorBox.style.display = DisplayStyle.Flex;
+                errorBox.Q<Label>().text = bucketResponse.ErrorMessage;
+                Debug.Log(bucketResponse.ErrorMessage);
+            }
```

Comment:
I've noticed this pattern a few times. Is there no common utility to handle displaying error messages? It's probably identical code in all spots

Reply:
Within the PR #111 We have implemented error boxes correctly, once this is merged, these will be fixed.